meson.build

```diff
@@ -371,12 +371,20 @@ if host_machine.endian() == 'big'
     conf_data.set10('WORDS_BIGENDIAN', true)
 endif
 
-# Non-4K memory page size is supported only for ppc64 at the moment.
-# TODO re-enable ppc dynrec while working on W^X stuff
-#      disabled because SVN r4424 broke compilation of ppc backends
-#if host_machine.cpu_family() in ['ppc64', 'ppc64le']
-#  conf_data.set('PAGESIZE', 65536)
-#endif
+# 64-bit Power ISA can run with either 4K or 64K page size.
+# (32-bit Power ISA and PowerPC are always 4K.)
+if host_machine.cpu_family() in ['ppc64', 'ppc64le']
+    pagesize_cmd = run_command('getconf', 'PAGESIZE', check: true)
+    if pagesize_cmd.returncode() != 0
+        error('''error executing getconf''')
+    else
+        pagesize = pagesize_cmd.stdout().strip().to_int()
+        if pagesize != 4096
+            warning('''using pagesize of '''+pagesize.to_string())
+            conf_data.set('PAGESIZE', pagesize)
```

Comment:
Suggest avoiding using `warning`, because a > 4K page size is fine within the scope of the `if host_machine.cpu_family() in ['ppc64', 'ppc64le']` condition.

Suggest `summary('Host page size', size-as-string)`, which shows up in the summary block at the end of `meson setup` phase:

```
dosbox-staging 0.81.0-alpha

  Build Summary
    Build type          : debug
    Install prefix      : /usr/local
    ...
    ManyMouse support   : Enabled
    Byte order          : little-endian
    Per-page W^X        : True
    CPU dynamic core    : optimized for x86/x86_64 (dyn-x86)
    CPU core target arch: X86_64
```

It's OK if the `summary()` is just added for this != 4096 scenario, just to highlight to the user at the end.

Reply:
Even better, @classilla!

Can do the summary at the end (value from the conf) so will also show up as 16K for Apple ARM64, too. Everbody can see this improvement now :-)

Reply:
done (hopefully in an acceptable fashion)